apps/server/src/modules/account/services/account.service.ts

```diff
@@ -68,7 +68,12 @@ export class AccountService extends AbstractAccountService {
 			idmReferenceId: ret.id,
 			password: accountDto.password,
 		};
-		const idmAccount = await this.executeIdmMethod(async () => this.accountIdm.save(newAccount));
+		const idmAccount = await this.executeIdmMethod(async () => {
+			this.logger.debug(`Saving account with accountID ${ret.id} ...`);
```

Comment:
shouldn't we use the new logger here? 
see https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/logging.html